docs/content/en/docs/about/under-the-hood/guarantees.md

```diff
@@ -15,12 +15,20 @@ see which components support specific guarantees.
 
 ### At-least-once
 
-The **at-least-once** delivery guarantee ensures that an [event] received by a Vector component is
-ultimately delivered at least once. While rare, it is possible for an event to be delivered more
-than once. See the [Does Vector support exactly-once delivery?](#faq-at-least-once) FAQ below).
+The **at-least-once** delivery guarantee ensures that an [event]
+received by a Vector component is ultimately delivered at least
+once. For a source, this indicates that it will wait for _all_ connected
+sinks to mark the event as delivered before acknowledging receipt of the
```

Comment:
I think we are still missing the nuance here that, if a disk buffer is configured on the sink, that it won't wait for the event to be propagated all the way to where-ever the sink is sending, but just until it is persisted in the disk buffer.

I think this nuance is important for users to be aware of. They may naively think that the source won't ack until the event is literally pushed downstream of Vector.

Reply:
Does 3790d24e3 work for you?

Reply:
That works, thanks!